physics/sgscloud_radpre.F90

```diff
@@ -90,7 +92,8 @@ subroutine sgscloud_radpre_run(    &
       real(kind=kind_phys), dimension(:,:), intent(inout) :: qr, qs, qg
       ! qci_conv only allocated if GF is used
       real(kind=kind_phys), dimension(:,:), intent(inout) :: qci_conv
-      real(kind=kind_phys), dimension(:,:), intent(in)    :: ud_mf
+      real(kind=kind_phys), dimension(:,:), intent(in)    ::ud_mf
+      real(kind=kind_phys), dimension(:,:), intent(in)    ::ud_mf_timeave, qci_conv_timeave
```

Comment:
minor detail but can we preserve the space between :: and the first variable?

Reply:
Fixed